src/components/nav/NavFooter.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div>
+    <footer class="footer">
+      <div class="container">
+        <div class="column" align="left">
+          <h3>Made at</h3>
+          <a href="https://www.khoury.northeastern.edu/">
+            <img class="khoury-logo" src="../../assets/khoury-logo.png" />
+          </a>
+        </div>
+        <div class="column" align="left">
+          <h3>Upcoming events</h3>
+          <router-link
+            to="../../events"
+            v-for="event in events.slice(0, 3)"
+            :key="event.title"
+          >
+            <p>{{ event.title }}</p>
+          </router-link>
+        </div>
+        <div class="column" align="left">
+          <h3>Find us on</h3>
+          <a
+            href="https://www.facebook.com/c4cneu/"
+            class="fa fa-facebook hvr-grow"
+          ></a>
+          <a
+            href="https://www.instagram.com/c4cneu/"
+            class="fa fa-instagram hvr-grow"
+          ></a>
+          <a
+            href="https://www.linkedin.com/company/code-4-community/about/"
+            class="fa fa-linkedin hvr-grow"
+          ></a>
+          <a href="https://c4cneu.slack.com/" class="fa fa-slack hvr-grow"></a>
+          <a
+            href="https://github.com/Code-4-Community"
+            class="fa fa-github hvr-grow"
+          ></a>
+          <a
+            href="mailto:c4cneu@gmail.com"
+            class="fa fa-envelope-o hvr-grow"
+          ></a>
+        </div>
+      </div>
+    </footer>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
```

Comment:
We need to add a mounted() function here to load the Events, because currently that is only being done when the Events page is visited (it is being loaded from jsonplaceholder now). Look at the mounted() function in the events page for reference and add that here.